docker-compose.yml

```diff
@@ -192,3 +194,74 @@ services:
         retries: 20
     smtp-server:
       image: mailhog/mailhog
+    document-management-api-gateway-web:
+      image: docker.artifactory.reform.hmcts.net/evidence/em-api-gateway-web:latest
+      environment:
+        - NODE_ENV=development
+        - PORT=8080
+        - EM_API_URL=http://evidence-management-api:8080
+        - IDAM_BASE_URL=http://idam-api:8080
+        - IDAM_S2S_URL=http://service-auth-provider-api:8080
+        - IDAM_SERVICE_KEY=AAAAAAAAAAAAAAAA
+        - IDAM_SERVICE_NAME=em_gw
+        - CORS_ORIGIN_METHODS=GET,POST,PUT,DELETE,OPTIONS
+        - CORS_ORIGIN_WHITELIST=http://evidence-management-api:8080
+        #      logging env vars
+        - ROOT_APPENDER=JSON_CONSOLE
+        - JSON_CONSOLE_PRETTY_PRINT=false
+        - REFORM_SERVICE_TYPE=node
+        - REFORM_SERVICE_NAME=em-api-gateway-web
+        - REFORM_TEAM=cc
+        - REFORM_ENVIRONMENT=docker
+        - XFWD=true
+      ports:
+        - 8080:8080
+      links:
+       - document-management-api
+       - idam-api
+       - service-auth-provider-api
+      depends_on:
+       - document-management-api
+       - idam-api
+       - service-auth-provider-api
+
+    document-management-api:
+      image: docker.artifactory.reform.hmcts.net/evidence/evidence-management-store:latest
+      command: --wait-for-database 30
+      environment:
+        - IDAM_SUPERUSER_EMAIL
+        - IDAM_SECRET_KEY
+        - GOV_NOTIFY_API_KEY
+        - SPRING_DATASOURCE_URL=jdbc:postgresql://evidence-database:5432/evidence
+        - SPRING_DATASOURCE_USERNAME=evidence
+        - SPRING_DATASOURCE_PASSWORD=evidence
+        - SPRING_PROFILES_ACTIVE=dev
+        - IDAM_TESTING_SUPPORT_ENABLED=true
+        - LOGGING_LEVEL_UK_GOV_HMCTS_IDAM=DEBUG
+        - IDAM_CLIENT_URL=http://idam-api:8080
+        - PROVIDER_SERVICE_CLIENT_URL=http://service-auth-provider-api:8080
+        - http_proxy=
+        - https_proxy=
+        - no_proxy=
+  #      logging env vars
+        - ROOT_APPENDER=JSON_CONSOLE
```

Comment:
you probably don't want json console in local environment, I would remove all the logging config form here except the logging level